yada-api/pom.xml

```diff
@@ -2,7 +2,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>com.novartis.opensource</groupId>
+    <groupId>com.novartis.opensource.aspirin</groupId>
```

Comment:
can we avoid this 'subgroup' ?
this is dicey because there is possibly Dave's original YADA 10.1.4-SNAPSHOT in some Maven repo somewhere, but I'm not sure which one you see when you test aspirin or mancala...